lib/actions.ex

```diff
@@ -40,7 +39,7 @@ defmodule EctoShorts.Actions do
       iex> EctoSchemas.Actions.all(EctoSchemas.Accounts.User)
       []
   """
-  @spec all(queryable :: query) :: schema_list
+  @spec all(queryable :: query) :: list()
```

Comment:
This can return a list of anything, since it's possible to add a select to the first argument, e.g.
`User.select_id()`